.github/ISSUE_TEMPLATE/enhancement-compression.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
suggestion: use consistent filename style, such as 'model-compression-bug-report' and 'model-compression-enhancement', or 'bug-report-model-compression' and 'enhancement-model-compression'

Reply:
ok!